artemis_cli

```diff
@@ -235,6 +283,7 @@ def main():
         cfg = yaml.safe_load(config_file)
 
     # alias commonly used config fields
+    general = cfg['general']
```

Comment:
@oberrich This was not added in config.yml or the changes not pushed (probably the latter)? It should also be caught (please enter general (what does it actually do?) in config.yml) and mentioned in README, if that's an action, the user should take.

Reply:
Good catch, config.yml is in .gitignore in order to not accidentally leak passwords